src/routes/faucet.ts

```diff
@@ -78,13 +81,15 @@ async function getNextBlockHash(wallet: Wallet, blockEmitter: BlockEmitter, bloc
 
       return nextBlock.hash
     }
+    logger.debug(`getNextBlockHash failed to get blockhash for blockNumber ${blockNumber}`)
```

Comment:
warn or error?

Reply:
Not sure. The error will is caught in the function that calls this and logged there. But I can make it error